### DISCUSSION TITLE
About the 'atomic_dipole.npy' file

### DISCUSSION THREAD

**Discussion by SeyongChoi** [deepmodeling/deepmd-kit#3209] [Q&A]
Hi, I have question about the 'atomic_dipole.npy' file format to use **dipole(polar) model** or **long-range DP model**.

I am wondering how to make the 'atomic_dipole.npy' file. 
I've checked the 'atomic_dipole.npy' file in `/examples/water_tensor/` as well, but I did not exactly understand what it included.

Is it the meaning of the dipole moment of each **atom** in the systems? or Wanniner centroid?

If it is, how do I get the atomic dipole moment? 
Does it come from the equation below? (ref. Phys. Chem. Chem. Phys., 2020, 22, 10592)
$\mu=e\sum_{i}Z_{i}r_{i}-2e\sum_{k}w_{k}$

or is it just dividing the total dipole moment vectors of systems like what I do get the DP model for energy?

And in the example folder, the 'coord.npy' file shape [80, 576] is not the same as the 'atomic_dipole.npy' file shape [80, 192] why it is different?

**Comment by Florian-Pabst** [comment]
Hi,
in the 'atomic_dipole.npy'  file you put vectors, for example the dipole moment vector of each atom (which you might get from Wannier centers) or, as in the case of the water example, the distance of the Wannier centroid from the O atom. This is why the 'atomic_dipole.npy' file has less entries than the 'coord.npy' file, because no vectors are assigned to the H atoms.

**Reply by SeyongChoi** [reply]
I'm late to check it, but thank you for your reply. 

I've got it about the shape problem of example files and using Wannier centroid information, in the case of the water examples.

However, I'm uncertain about the equation for getting the dipole moment vector of each atom using Wannier centers. 
Is the equation I mentioned one of the methods for obtaining it?
